platform/template/docker-syncd-base.mk

```diff
@@ -0,0 +1,35 @@
+# docker image for syncd
+
+
+DOCKER_SYNCD_BASE_STEM = docker-syncd-$(DOCKER_SYNCD_PLATFORM_CODE)
+DOCKER_SYNCD_BASE = $(DOCKER_SYNCD_BASE_STEM).gz
+DOCKER_SYNCD_BASE_DBG = $(DOCKER_SYNCD_BASE_STEM)-$(DBG_IMAGE_MARK).gz
+
+$(DOCKER_SYNCD_BASE)_PATH = $(PLATFORM_PATH)/docker-syncd-$(DOCKER_SYNCD_PLATFORM_CODE)
+$(DOCKER_SYNCD_BASE)_DEPENDS += $(SYNCD) $(PYTHON_SDK_API)
+$(DOCKER_SYNCD_BASE)_PYTHON_DEBS += $(MLNX_SFPD)
```

Comment:
these are mellanox specific code, should not be in syncd-base.

Reply:
moved dependent stuff  off the template.